backend/jobsearch/src/main/kotlin/com/jobsearch/dto/JobRequestDTO.kt

```diff
@@ -0,0 +1,35 @@
+package com.jobsearch.dto
+
+import jakarta.validation.constraints.Max
+import jakarta.validation.constraints.Min
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotNull
+
+data class JobRequestDTO(
+    val id: Int?,
+
+    @field:Min(value = 1900, message = "Start year must be reasonable")
+    @field:Max(value = 2100, message = "Start year must be reasonable")
+    val startYear: Int,
+
+    @field:Min(value = 1, message = "Start month must be between 1 and 12")
+    @field:Max(value = 12, message = "Start month must be between 1 and 12")
+    val startMonth: Int,
+
+    @field:Min(value = 1900, message = "End year must be reasonable")
+    @field:Max(value = 2100, message = "End year must be reasonable")
+    val endYear: Int,
+
+    @field:Min(value = 1, message = "End month must be between 1 and 12")
+    @field:Max(value = 12, message = "End month must be between 1 and 12")
+    val endMonth: Int,
```

Comment:
Please use the [Date](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.js/-date/) for `start` and `end`